source/_patterns/04-pages/msa.mustache

```diff
@@ -24,21 +24,25 @@
       </div>
 
 
-    <div class="wrapper">
+    <div class="wrapper wrapper--listing wrapper--listing-selector">
```

Comment:
There is no need for the wrapper--listing or wrapper--listing selector classes.

Reply:
> Do we need the ability to alter the url when we toggle between the view selector options?

I got the impression by our conversation earlier this week that we need to keep the url logic. Also I can see that on prod it is used to jump on a parent section when the first tab is clicked (see the image below). Please correct me if I'm wrong, I will be happy if you advise me.

![image](https://user-images.githubusercontent.com/111341278/202762770-9ddb1a36-7a74-47bc-98c2-b66013ff2fb6.png)